Rakefile

```
@@ -214,7 +214,7 @@ namespace :history do
     Dir.chdir( File.expand_path(File.dirname(__FILE__)) )
     output = `bundle exec ruby history.rb .`
     puts output
-    if !/success/.match?(output)
+    if !output.include?(success)
```

```suggestion
    if !output.include?(''success')
```

One gotcha though is the old form was nil-safe, so this works `/success/.match?(nil)` but `nil.include?('success')` doesn't

---

So should it use the safe navigator?
```suggestion
    if !output&.include?('success')
```